### PR TITLE
enh(getting started) Monitor a MySQL or MariaDB database

### PR DESCRIPTION
**Pull request by cedricSMLA**
## Description

Rewrite Monitor a MySQL or MariaDB database on a dedicated page. fr and en

## Target version

- [ ] 20.10.x
- [ ] 21.04.x
- [ x] 21.10.x
- [ x] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/getting-started/mysql_tuto.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Afin de pouvoir collecter les informations nécessaires à partir de la base de données, un utilisateur de la base de données disposant de privilèges spécifiques sera nécessaire. Si la base que vous souhaitez superviser est une base Centreon, un utilisateur dédié existe déjà (il s'appelle **centreon**). Dans le cas contraire, commencez par créer un nouvel utilisateur : connectez-vous à votre base de données, puis exécutez la commande suivante :
+
+```shell
+CREATE USER 'username'@'IP_POLLER' IDENTIFIED BY 'password';
+```
+
+Puis, que votre base soit une base Centreon ou non, exécutez la commande suivante :
+
+```shell
+grant ALL PRIVILEGES on *.* to 'centreon'@'IP_POLLER' identified by 'password'
```
Comment (pkriko) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/getting-started/mysql_tuto.md:19]: GRANT ALL PRIVILEGES ?
Looks like kind overkill and not secure :sweat: 
Only SELECT right should be granted.
not necessary to precise again the password.

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ yum install centreon-plugin-Applications-Databases-Mysql
 In order to be able to collect the needed information from the database a
 database user with specific privileges is required:
 
-    # grant usage on *.* to 'centreon'@'pollerip' identified by 'password'
+```shell
+grant usage on *.* to 'centreon'@'pollerip' identified by 'password'
+```
 
 ### MariaDB, perl-dbi, perl-dbd-mysql
```
Comment (pkriko) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/procedures/applications-databases-mysql.md:28]: Why you change a monitoring proc ?

File: versioned_docs/version-21.10/getting-started/mysql_tuto.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+```shell
+CREATE USER 'username'@'IP_POLLER' IDENTIFIED BY 'password';
+```
+
+Then, whether your database is a Centreon database or not, run the following command:
+
+```shell
+grant ALL PRIVILEGES on *.* to 'username'@'IP_POLLER' identified by 'password'
+```
```
Comment (pkriko) [inline versioned_docs/version-21.10/getting-started/mysql_tuto.md:20]: Please use GRANT SELECT and respect the CAPITAL letter :)
Something like this:
GRANT SELECT ON . TO 'centreon'@'pollerip';

Not necessary to precise again the password ;-)

**Review by pkriko — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/procedures/applications-databases-mysql.md (hunk 2 above)
Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs/version-21.10/integrations/plugin-packs/procedures/applications-databases-mysql.md:28]: I did, to make the markdown more consistent

**Review by cg-tw — DISMISSED** [review]

**Review by pkriko — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**